File: task3/lvhaoyu/shuapiao.php
<?php
    set_time_limit(0);
    $i=0;
    $url='http://today.hit.edu.cn/commend/114154_1.htm';
    do{
      $a = rand(1, 255);
      $b = rand(1, 255);
      $c = rand(1, 50);
      $d = rand(1, 255);
      $ip = "$a.$b.$c.$d";
      $ch = curl_init();
      curl_setopt($ch, CURLOPT_HTTPHEADER, array('X-FORWARDED-FOR:'.$ip,'CLIENT-IP:'.$ip));
      curl_setopt($ch, CURLOPT_HEADER,1);
      curl_setopt($ch, CURLOPT_URL, $url);
      sleep(2);
      $i++;
      $out = curl_exec($ch);
      curl_close($ch);
    }while($i<=100);
?>

File: task3/wethug/shuapiao.java
import java.net.*;
import java.util.*;

public class shuapiao
{
	public static void main(String[] args)
	{    
		try{
			for(int i=1;i<=100;i++)
			{
				//获取随机IP值
				Random getRandom=new Random();
				int ip1=getRandom.nextInt(255);
				int ip2=getRandom.nextInt(255);
				int ip3=getRandom.nextInt(255);
				int ip4=getRandom.nextInt(255);
				String ip=ip1+"."+ip2+"."+ip3+"."+ip4;
				
				//创建一个url对象			
				URL url=new URL("http://today.hit.edu.cn/commend/113987_1.htm");
				
				//从网络中获取网页数据
				HttpURLConnection con = (HttpURLConnection)url.openConnection();
				
				//设置请求属性
				con.setRequestProperty("X-Forwarded-For",ip);
				
				//发送请求
				con.getHeaderFields();					
				
				//延迟一秒
				Thread.sleep(1000);  
			}
		}catch (Exception e){
			e.printStackTrace();	
		}
	}
}
